### PR TITLE
[awattar] Fix that allows bridge to be added via UI.

### PR DESCRIPTION
**Pull request by dougculnane**
As per discussion in the forum: https://community.openhab.org/t/awattar-binding-beta-and-discussion/110497/70

the awatar bridge can not be added as a thing with the UI 3.  This fixes (or works around) the problem.

Signed-off-by: Doug Culnane <doug@culnane.net>

### CODE REVIEW

File: bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/bridge-types.xml [hunk 1]
```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<thing:thing-descriptions bindingId="openweathermap"
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/bridge-types.xml:2]: The reason it didn't work is that the `bindingId` is wrong in this file

Comment (dougculnane) [reply bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/bridge-types.xml:2]: @Hilbrand OK.  That is the real problem then. My fix was more of a work around that worked.  

What is better, my fix or changing the bindingId?

Comment (Hilbrand) [reply bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/bridge-types.xml:2]: In general there is no best choice. Either solution is ok. In this case it probably makes more sense to have everything in one file as it's only a small set of things. I would say your fix is ok.

**Review by Hilbrand — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/bridge-types.xml (hunk 1 above)
Comment (Wolfgang1966) [reply bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/bridge-types.xml:2]: Sorry for the copy-paste error, was my bad :-(

**Comment by Wolfgang1966** [comment]
LGTM

**Review by Wolfgang1966 — APPROVED** [review]

File: bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/bridge-types.xml (hunk 1 above)
Comment (dougculnane) [reply bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/bridge-types.xml:2]: @Wolfgang1966 no problem it gave me an easy fix (work around).

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**